crates/uv/tests/it/lock.rs

```diff
@@ -2360,6 +2360,15 @@ fn lock_conflicting_extra_basic() -> Result<()> {
     exit_code: 2
     ----- stdout -----
 
+    ----- stderr -----
+    error: extra `project1`, extra `project2` are incompatible with the declared conflicts: {`project[project1]`, `project[project2]`}
```

Comment:
I know this isn't from this pull request, but can we call these extras something other than `project` e.g., `extra1` and `extra2` for clarity? I changed these in the documentation but didn't realize there were tests with the same cases.

Reply:
Yeah sure.

Reply:
Per other PR, I'll do this in a dedicated change, since a lot of other tests do this too.